lib/mutant/mutator/node/regexp/greedy_zero_or_more.rb

```diff
@@ -0,0 +1,24 @@
+module Mutant
+  class Mutator
+    class Node
+      module Regexp
+        # Mutator for greedy zero-or-more quantifier, `*`
+        class GreedyZeroOrMore < Node
+          handle(:regexp_greedy_zero_or_more)
+
+          children :min, :max, :subject
+
+          # Emit mutations
+          #
+          # Replace `/a*/` with `/a+/`
+          #
+          # @return [undefined]
+          def dispatch
+            emit(s(:regexp_greedy_one_or_more, *children))
+            emit_subject_mutations
```

Comment:
k. I assumed a similar AST design than we have in ruby. This should be left as is for now.

Curious why we have both `min` and `max` in the first place, when the node defines matching numbers already by type.